GDJS/GDJS/IDE/ExporterHelper.h

```diff
@@ -243,19 +246,34 @@ struct PreviewExportOptions {
    * \brief Set the template slug that was used to create the project.
    */
   PreviewExportOptions &SetProjectTemplateSlug(
-      const gd::String& projectTemplateSlug_) {
+      const gd::String &projectTemplateSlug_) {
     projectTemplateSlug = projectTemplateSlug_;
     return *this;
   }
 
   /**
    * \brief Set the source game id that was used to create the project.
    */
-  PreviewExportOptions &SetSourceGameId(const gd::String& sourceGameId_) {
+  PreviewExportOptions &SetSourceGameId(const gd::String &sourceGameId_) {
     sourceGameId = sourceGameId_;
     return *this;
   }
 
+  /**
+   * \brief Set the capture options to be used for taking screenshots or videos
+   * of the preview.
+   */
+  PreviewExportOptions &AddScreenshotCapture(const int &timing,
```

Comment:
Passing a reference to a int is unusual, it's only when you want to modify it. Passing a const reference means that you can't modify it, but an int in memory is basically the same size as a reference (= pointer).
tl;dr: int can always be passed by value.

```suggestion
  PreviewExportOptions &AddScreenshotCapture(int timing,
```